src/java/com/zimbra/qa/soap/SoapTestCore.java

```diff
@@ -1,3301 +1,3315 @@
+    		} else {
+    			if ( !core.mCurrTestCase.getBugIDs().equals("NO_BUGS_LOGGED") ) {
+    				core.summaryResults.append(" (Fixed Bug: #" + core.mCurrTestCase.getBugIDs() + ")");
+    			}
+    		}
+    		core.summaryResults.append(System.lineSeparator());
+
+        	if ( testStep.testFailed() ) {
+        		core.mCurrTestCase.addTestStepFailure(1);	// TestCase Failed!
+            }
+
+		}
+    	
+    }
+    private void getFile(File inputFile){
+        if(!inputFile.isDirectory()){
+            toBeIgnored.add(inputFile);
+        } else {
+            for (File currentFile : inputFile.listFiles(xmlFileFilter)){
+                getFile(currentFile);
+            }
+        }
+    }
+
+    private void getIgnoredTests(){
+        String line = null;
+        FileReader fr;
+
+        try {
+            if (SoapTestMain.globalProperties.getProperty("server.zimbrax").equals("true")) {
+                fr = new FileReader(SoapTestCore.rootZimbraQA + "/conf/skipped-tests-zimbrax.txt");
+            } else {
+                fr = new FileReader(SoapTestCore.rootZimbraQA + "/conf/skipped-tests.txt");
+            }
+
+            BufferedReader bufferedReader = new BufferedReader(fr);
+            while ((line = bufferedReader.readLine()) != null) {
+                File enlistedTestCases = new File(line);
+                if (!enlistedTestCases.isDirectory()) {
+                    toBeIgnored.add(enlistedTestCases);
+                } else {
+                    getFile(enlistedTestCases);
+                }
+            }
+            bufferedReader.close();
+
+        } catch (FileNotFoundException e1) {
+            e1.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    
+    /**
+     * Execute tests in an input XML file.
+     * 
+     * @param inputFile
+     * @return
+     * @throws DocumentException
+     * @throws HarnessException
+     * @throws SoapFaultException
+     * @throws IOException
+     * @throws HarnessException 
+     * @throws HarnessException 
+     */
+    public void runTestFile(File inputFile) throws DocumentException, IOException, HarnessException {
+
+        try {
+            if(TestProperties.testProperties.getProperty("ignore.files").equals("true")){
+                getIgnoredTests();
+                if(toBeIgnored.contains(inputFile)){
+                    return;
+                }
+            }
+        } catch (HarnessException e1) {
+            // TODO Auto-generated catch block
+            e1.printStackTrace();
+        }
+        FileAppender fileAppender = null;
+        mTraceLogger = LogManager.getLogger("zimbra.qa.trace");
+        mResultLogger = LogManager.getLogger("zimbra.qa.trace.result");
+
+        //Additivity has been set to false in order to avoid duplicate log being printed in the console during execution
+        //mTraceLogger.setAdditivity(false);

```

Comment:
is there a change in this config for additivity or is it default false now in the update. Can this be remove if not requried?

Reply:
we have defined it in the log4j properties now.